# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/DocumentChannelParent.h
# Commit: 8575cd92e1d8
# Full Hash: 8575cd92e1d8cd783e6cca3f6feadef11f0c4cc8
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 8: Add list of open channels from OngoingEarlyHints to RedirectToRealChannelArgs r=necko-reviewers,valentin,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161179
# ==============================================================================

diff -r ce8f0a8fa688 -r 8575cd92e1d8 netwerk/ipc/DocumentChannelParent.h
--- a/netwerk/ipc/DocumentChannelParent.h	Fri Dec 02 16:15:55 2022 +0000
+++ b/netwerk/ipc/DocumentChannelParent.h	Fri Dec 02 16:15:56 2022 +0000
@@ -16,6 +16,8 @@
 }
 namespace net {
 
+class EarlyHintConnectArgs;
+
 /**
  * An actor that forwards all changes across to DocumentChannelChild, the
  * nsIChannel implementation owned by a content process docshell.
@@ -52,7 +54,8 @@
   RedirectToRealChannel(
       nsTArray<ipc::Endpoint<extensions::PStreamFilterParent>>&&
           aStreamFilterEndpoints,
-      uint32_t aRedirectFlags, uint32_t aLoadFlags);
+      uint32_t aRedirectFlags, uint32_t aLoadFlags,
+      nsTArray<EarlyHintConnectArgs>&& aEarlyHints);
 
   virtual ~DocumentChannelParent();
 